classes/local/entities/enrollment.php

```diff
@@ -91,9 +91,19 @@ public function get_user_entity(): ?user {
         // Fetch the user details.
         $userref = $this->get('user');
 
+        // Handle both API format (object with sourcedId) and CSV format (direct string)
```

Comment:
This comment seems a bit strange. CSV format should still be a row, with sourcedId as a field. Can this be clarified?